sdk/script/account-manager.ts

```diff
@@ -334,21 +339,11 @@ export class AccountManager {
     }
 
     private attachCredentials(request: superagent.Request<any>): void {
-        request.set(`User-Agent`, this._userAgent);
         request.set(`Accept`, `application/vnd.code-push.${AccountManager.API_VERSION}+json`);
         request.set(`Authorization`, `Bearer ${this._accessKey}`);
-    }
-
-    private generateAccessKey(): Promise<string> {
-        return this.getAccountInfo()
-            .then((account: Account) => {
-                var accessKey = crypto.randomBytes(21)
-                    .toString(`base64`)
-                    .replace(/\+/g, `_`)  // URL-friendly characters
-                    .replace(/\//g, `-`)
-                    .concat(account.email);
-
-                return accessKey;
-            });
+        if (this._userAgent) {
+            request.set(`User-Agent`, this._userAgent);
```

Comment:
Above you have great examples of template literals used effectively.  However, if string interpolation is not required, use plain old string literals.  This feedback applies globally.